.github/workflows/qa-deploy.yaml

```
@@ -162,3 +162,21 @@ jobs:
           repository: TeleVet/e2e-tests
           event-type: core-api-automated-test-isolated
           client-payload: '{"ref": "${{ github.event.pull_request.head.ref }}", "sha": "${{ github.event.pull_request.head.sha }}", "repository": "core-api", "user": "${{ github.actor }}", "qa_lane": "${{ steps.select-qa-env.outputs.env_name }}"}'
+
+      - name: Run Core Jobs Automation Tests
+        uses: peter-evans/repository-dispatch@v1
+        if: ${{ github.repository == 'TeleVet/core-jobs' }}
```

this one is slightly tricky since we do multiple qa-deploys on every code change
```suggestion
        if: ${{ github.repository == 'TeleVet/core-jobs' && inputs.app_name == 'core-jobs-sqs-model-changed' }}
```